Windows test fail

https://ci.ipfs.team/blue/organizations/jenkins/IPFS%2Fgo-ipld-git/detail/master/1/tests

```
tests / windows / TestObjectParse – go-ipld-git
Error
Failed
Stacktrace
panic: runtime error: index out of range
c:/jenkins/tools/org.jenkinsci.plugins.golang.GolangInstallation/1.10.2/src/testing/testing.go:742 +0x2a4
c:/jenkins/tools/org.jenkinsci.plugins.golang.GolangInstallation/1.10.2/src/runtime/panic.go:502 +0x237
c:/jenkins/workspace/IPFS_go-ipld-git_master-66LUBMIUPM6OP6TSZ2OKJLPQ4APLAPM3ANAZQ7FC2VGOGWDQODYQ/src/github.com/ipfs/go-ipld-git/git_test.go:31 +0xb75
c:/jenkins/tools/org.jenkinsci.plugins.golang.GolangInstallation/1.10.2/src/path/filepath/path.go:357 +0x409
c:/jenkins/tools/org.jenkinsci.plugins.golang.GolangInstallation/1.10.2/src/path/filepath/path.go:381 +0x2c9
c:/jenkins/tools/org.jenkinsci.plugins.golang.GolangInstallation/1.10.2/src/path/filepath/path.go:381 +0x2c9
c:/jenkins/tools/org.jenkinsci.plugins.golang.GolangInstallation/1.10.2/src/path/filepath/path.go:403 +0x10d
c:/jenkins/workspace/IPFS_go-ipld-git_master-66LUBMIUPM6OP6TSZ2OKJLPQ4APLAPM3ANAZQ7FC2VGOGWDQODYQ/src/github.com/ipfs/go-ipld-git/git_test.go:22 +0xa7
c:/jenkins/tools/org.jenkinsci.plugins.golang.GolangInstallation/1.10.2/src/testing/testing.go:777 +0xd7
c:/jenkins/tools/org.jenkinsci.plugins.golang.GolangInstallation/1.10.2/src/testing/testing.go:824 +0x2e7
```

--- a/git_test.go
+++ b/git_test.go
@@ -28,7 +28,8 @@ func TestObjectParse(t *testing.T) {
 			return nil
 		}
 
-		parts := strings.Split(path, "/")
+		parts := strings.Split(path, string(filepath.Separator))
+
 		dir := parts[len(parts)-2]
 		if dir == "info" || dir == "pack" {
 			return nil
@@ -360,7 +361,7 @@ func BenchmarkRawData(b *testing.B) {
 				return nil
 			}
 
-			parts := strings.Split(path, "/")
+			parts := strings.Split(path, string(filepath.Separator))
 			if dir := parts[len(parts)-2]; dir == "info" || dir == "pack" {
 				return nil
 			}
@@ -393,7 +394,7 @@ func BenchmarkCid(b *testing.B) {
 				return nil
 			}
 
-			parts := strings.Split(path, "/")
+			parts := strings.Split(path, string(filepath.Separator))
 			if dir := parts[len(parts)-2]; dir == "info" || dir == "pack" {
 				return nil
 			}

